modules/util/rebind.js

```diff
@@ -1,8 +1,16 @@
 // Copies a variable number of methods from source to target.
-export function utilRebind(target, source) {
-    var i = 1, n = arguments.length, method;
-    while (++i < n) {
-        target[method = arguments[i]] = d3_rebind(target, source, source[method]);
+/**
+ * @template T
+ * @template S
+ * @template {keyof S} Args
+ * @param {T} target
+ * @param {S} source
+ * @param {...Args} args
+ * @returns {T & Pick<S, Args>}
+ */
+export function utilRebind(target, source, ...args) {
```

Comment:
this had to be refactored to use [spread arguments](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) instead of the [`arguments` keyword](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments), because JSDoc doesn't support the `arguments` keyword.

there's a new unit test to confirm that [`this`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this) is still correctly bound.